static/style.css

```diff
@@ -1,4 +1,8 @@
 
+.container {
```

Comment:
Is there a reason for this empty css goup? There might be and I just don't know it...but it looks odd.